localization/ru_RU.inc

```diff
@@ -1,6 +1,5 @@
 <?php
-
 $labels = array();
-
-$labels['internal'] = "Письмо было внутренним";
-$labels['unencrypted'] = "Сообщение получено по незашифрованному соединению!";
\ No newline at end of file
+$labels['internal'] = 'Почта была внутренней';
+$labels['unencrypted'] = 'Сообщение получено по незашифрованному соединению!';
+?>
```

Comment:
@liuch can you validate this translation ?

Reply:
> Or maybe localization/uk_UA.inc ?

What do you mean?